### DISCUSSION TITLE
Make it easier for content editors to mark entry as homepage

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#13899] [Ideas]
Right now, we need to use `__home__` slug for that. This is not ideal, because it required editors to have knowledge of that "magical" string. It would be better if there was some kind of checkbox, like one we have when setting up uri of section.

**Comment by janhenckens** [comment]
There's a checkbox for that when you create a Single:
![Screenshot 20231103-121625-dF19ty60](https://github.com/craftcms/cms/assets/755428/b000c4f1-f477-4457-8b7d-5c44716c4215)

(which could probably also use an info bubble to add a bit more context)

**Reply by piotrpog** [reply]
I am aware of that, but i am talking of making one of the pages in channel type section a homepage.
